components/eam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -1046,6 +1047,15 @@ subroutine micro_p3_tend(state, ptend, dtime, pbuf)
        ! ccn is uniformly distributed throughout the cell, but P3 computes in-cloud values assuming cell-averages are comprised 
        ! of zero values outside cloud. Preemptively multiplying by cldfrac here is needed to get the correct in-cloud ccn value in P3
        nccn_prescribed = ccn_trcdat*cld_frac_l
+       ! SPA produced ccn is in units of #/cm3 rather than #/kg, so we need to
```

Comment:
Isn't the point that we want to change the SPA input files so CCN is in #/kg and we don't need to do this calculation either in C++ or in F90????

Reply:
Yes. If we have the SPA files ready for SCREAMv0 to run without this conversion than we can add them to this PR and remove this line.  I meant for this PR to clean up the interface without impacting SCREAMv0 runs, while we wait for the SPA data.

If we want, we can wait for the data to come available before integrating this PR.

Reply:
Aren't the new SPA files available now so we can delete this?